specs/extensions/openapi-extensions.md

```diff
@@ -0,0 +1,130 @@
+# OpenAPI Extensions
```

Comment:
this should probably be in our public documentation (once we released) and/or [here](https://github.com/microsoft/OpenAPI/tree/main/extensions)

Reply:
Yes, and I would say "and" for having it in both places!